### PR TITLE
Fix config error in xml.tmLanguage.json #184852

### LINKED ISSUE #184852: TextMate config error in /extensions/xml/syntaxes/xml.tmLanguage.json
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: 1.79
- OS Version: Windows

Steps to Reproduce:
The file https://github.com/microsoft/vscode/blob/main/extensions/xml/syntaxes/xml.tmLanguage.json contains a syntax/schema/config error. The "end" and "name" property are added add the wrong level.

https://github.com/microsoft/vscode/blob/7ff66b360d5f1cdbdbfa8e10842457e38117713d/extensions/xml/syntaxes/xml.tmLanguage.json#L352-L364

This was discovered when trying to parse the TextMate file using Eclipse TM4E.

### PR DESCRIPTION
**Pull request by sebthom**
This PR fixes a config issue in the TextMate config file /extensions/xml/syntaxes/xml.tmLanguage.json
where two properties are added at the wrong indention level. This PR places the properties at the correct level.

Fixes https://github.com/microsoft/vscode/issues/184852

### CODE REVIEW

File: extensions/xml/syntaxes/xml.tmLanguage.json [hunk 1]
```diff
@@ -357,9 +357,9 @@
 						"0": {
 							"name": "punctuation.definition.comment.xml"
 						},
-						"end": "--%>",
-						"name": "comment.block.xml"
 					}
```
Comment (awesomelewis2007) [inline extensions/xml/syntaxes/xml.tmLanguage.json:360]: Doesn't this need a comma 
```suggestion
					},
```

**Review by 0x4248 — CHANGES_REQUESTED** [review]

**Review by precha1989 — APPROVED** [review]
good

**Comment by sebthom** [comment]
@alexdima could you please review this PR? Thanks in advance!

**Comment by sebthom** [comment]
@alexr00 Why did you close the PR and not just merge it to fix the issue for now? It is a trivial change and you don't yet have another repo to source the grammar from.

**Pull request closed without merging**